x-pack/plugins/audit_trail/server/plugin.ts

```diff
@@ -50,7 +50,19 @@ export class AuditTrailPlugin implements Plugin {
       getSpaceId: deps.spaces?.spacesService.getSpaceId,
     };
 
-    this.event$.subscribe(({ message, ...other }) => this.logger.debug(message, other));
+    if (deps.security) {
+      let previousSubscription: Subscription;
+      deps.security.license.features$.subscribe(({ allowAuditLogging }) => {
```

Comment:
can it be expressed via Observables?
```js
combineLatest([this.event$.asObservable(), deps.security.license.features$]).pipe(
  filter(([, license]) => license.allowAuditLogging),
  tap(([{ message, ...other }]) => this.logger.debug(message, other))
);
```

Reply:
Close :)
I had to subscribe at the end but otherwise worked nicely